x-govuk/components/secondary-navigation/template.njk

```diff
@@ -0,0 +1,11 @@
+<nav class="x-govuk-secondary-navigation {%- if params.classes %} {{ params.classes }}{% endif -%}" aria-label="{{ params.visuallyHiddenTitle or "Secondary Menu" }}" {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
```

Comment:
What about `aria-labelled` by? I think that’d work well for the second example used in the documentation for this component.